polkadot.json

```
@@ -7,68 +7,68 @@
+        "details": "Polkadot is super adaptable, so developers can choose how they want to build their own blockchain. You can use prebuilt templates, design custom applications, or even re-architect the low level system! The possibilities are endless!"
       },
       {
         "id": 7,
         "imageUrl": "https://i.imgur.com/kQKhtLG.png",
         "title": "The future of blockchain",
-        "details": "Consider legacy blockchains to be small islands separated from one another. Each island has its own set of resources and restrictions, making communication difficult. That's similar to traditional blockchains! They're all separate from one another, with their own set of laws and constraints, making it difficult to share information or collaborate. Polkadot, on the other hand, functions as a bridge between these 'islands,' uniting them so they may work more efficiently and quickly."
+        "details": "Consider traditional blockchains to be small islands separated from one another. Each island has its own needs, making scaling and communication difficult. Polkadot, on the other hand, functions as a bridge, security team, and resource provider for all of these 'islands', empowering them to achieve their highest potential."
       },
       {
         "id": 8,
         "choices": [
-          "By allowing different parachains to collaborate and transfer data without much delay",
-          "By using advanced encryption techniques",
-          "By limiting the number of transactions that can be processed at once"
+          "Polkadot provides security to the applications connected to it",
+          "Polkadot provides communication channels and interoperability between applications",
+          "Polkadot provides a scalable and flexible platform for developing applications",
+          "All of the above"
         ],
-        "correctChoice": "By allowing different parachains to collaborate and transfer data without much delay",
-        "title": "How does Polkadot improve blockchain transactions?"
+        "correctChoice": "All of the above",
+        "title": "How does Polkadot help other blockchains and Web3 applications?"
       },
       {
         "id": 9,
         "choices": [
-          "Improved speed and efficiency of blockchain transactions",
-          "Customizable blockchains that allow developers to design specific functionalities for their chain",
-          "Ability to connect different blockchains and transfer assets easily",
-          "All of the above"
+          "Polkadot uses expensive hardware to vertically scale their blockchain",
+          "Polkadot parallelizes incoming transactions and shards data across its applications",
+          "Polkadot only secures applications when there is a dispute"
```

Elaborating a bit, as the original option says less on how it scales

```suggestion
          "Polkadot optimistically secures applications and checks for transaction validity only when there is a dispute"
```